<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNS</class>
 <widget class="QMainWindow" name="CNS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 1px solid;
border-radius: 30%;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_6">
      <property name="styleSheet">
       <string notr="true">border: 0;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>img/icons8-lupa-48.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>CNS - Consulta Agilizada</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{
background: #333;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QToolBox" name="toolBox">
               <property name="styleSheet">
                <string notr="true">QLabel{
background: None;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="page">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>332</width>
                  <height>298</height>
                 </rect>
                </property>
                <attribute name="label">
                 <string>Parametros</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="9" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Gerente</string>
                   </property>
                  </widget>
                 </item>
                 <item row="9" column="1">
                  <widget class="QLabel" name="label_5">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Regional</string>
                   </property>
                  </widget>
                 </item>
                 <item row="10" column="1">
                  <widget class="QComboBox" name="boxRegional"/>
                 </item>
                 <item row="3" column="0" colspan="2">
                  <widget class="QComboBox" name="boxCr"/>
                 </item>
                 <item row="10" column="0">
                  <widget class="QComboBox" name="boxGerente"/>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDateEdit" name="dateFinal">
                   <property name="buttonSymbols">
                    <enum>QAbstractSpinBox::NoButtons</enum>
                   </property>
                   <property name="correctionMode">
                    <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Servi√ßo</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Contrato - CR</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QDateEdit" name="dataInicial">
                   <property name="buttonSymbols">
                    <enum>QAbstractSpinBox::NoButtons</enum>
                   </property>
                   <property name="correctionMode">
                    <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0" colspan="2">
                  <widget class="QLabel" name="lblPeriodo">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>5</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Periodo</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="11" column="0">
                  <widget class="QPushButton" name="pushButton">
                   <property name="styleSheet">
                    <string notr="true">QPushButton{
border: 1px solid #333;
color: white;
border-radius: 20px;

}
</string>
                   </property>
                   <property name="text">
                    <string>Limpar</string>
                   </property>
                  </widget>
                 </item>
                 <item row="11" column="1">
                  <widget class="QPushButton" name="pushButton_2">
                   <property name="text">
                    <string>Consultar</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0" colspan="2">
                  <widget class="QComboBox" name="boxServico"/>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_2">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>332</width>
                  <height>298</height>
                 </rect>
                </property>
                <attribute name="label">
                 <string>Configura√ß√µes</string>
                </attribute>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="styleSheet">
             <string notr="true">QFrame{
background: #333;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QTableWidget" name="tableWidget">
               <column>
                <property name="text">
                 <string>Numero</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Tarefa</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Colaborador</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Gerente</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>InicioReal</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>TerminoReal</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Disponibiliza√ß√£o</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Local</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Pergunta</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Resposta</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Longitude</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Latitude</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Estrutura Completa</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
